src/tests/ftest/launch.py

```diff
@@ -41,6 +41,8 @@
 from ClusterShell.NodeSet import NodeSet
 from ClusterShell.Task import task_self
 
+from util.general_utils import stop_processes
```

Comment:
(pylint-no-name-in-module) No name 'general_utils' in module 'util'
(pylint-import-error) Unable to import 'util.general_utils'
(pylint-unused-import) Unused stop_processes imported from util.general_utils